spec/spec_params.rb

```diff
@@ -27,7 +27,7 @@ module OptimizelySpec
     'version' => '2',
     'events' => [{
       'key' => 'test_event',
-      'experimentIds' => ['111127'],
+      'experimentIds' => %w[111127 122230],
```

Comment:
why %w ?

Reply:
It's ruby convention for ['111127', '122230'], also rubocop suggestion.